prow/config/config.go

```diff
@@ -942,6 +942,23 @@ type Gerrit struct {
 	AllowedPresubmitTriggerReRawString string          `json:"allowed_presubmit_trigger_re,omitempty"`
 }
 
+func (g *Gerrit) DefaultAndValidate() error {
```

Comment:
The changes to this file just pull out the defaulting and validation logic for the Gerrit config struct so that we can reuse it in unit tests.